wellsfargoAPI.py

```diff
@@ -0,0 +1,450 @@
+                password_field = driver.find_element(By.XPATH, "//*[@id='j_password']")
+                type_slowly(password_field, account[1])
+
+                login_button = WebDriverWait(driver, 20).until(
+                    EC.element_to_be_clickable(
+                        (By.CSS_SELECTOR, ".Button__modern___cqCp7")
+                    )
+                )
+                login_button.click()
+                WebDriverWait(driver, 20).until(check_if_page_loaded)
+                print("=====================================================\n")
+            except TimeoutException:
+                print("TimeoutException: Login failed.")
+                return False
+            WELLSFARGO_obj.set_logged_in_object(name, driver)
+            try:
+                auth_popup = WebDriverWait(driver, 10).until(
+                    EC.presence_of_element_located(
+                        (By.CSS_SELECTOR, ".ResponsiveModalContent__modalContent___guT3p")
+                    )
+                )
+                auth_list = auth_popup.find_element(
+                    By.CSS_SELECTOR,
+                    ".LineItemLinkList__lineItemLinkList___Dj6vb"
+                )
+                li_elements = auth_list.find_elements(By.TAG_NAME, "li")
+                for li in li_elements:
+                    if account[2] in li.text:
+                        li.click()
+                        break
+                print("Clicked on phone number")
+                # Get the OTP code from the user
+                if botObj is not None and loop is not None:
+                    code = asyncio.run_coroutine_threadsafe(
+                        getOTPCodeDiscord(botObj, name, timeout=300, loop=loop),
+                        loop,
+                    ).result()
+                else:
+                    code = input("Enter security code: ")
+                code_input = WebDriverWait(driver, 20).until(
+                    EC.presence_of_element_located(
+                        (By.ID, "otp")
+                    )
+                )
+                code_input.send_keys(code)
+                WebDriverWait(driver, 10).until(
+                    EC.element_to_be_clickable(
+                        (By.XPATH, "//button[@type='submit']")
+                    )
+                ).click()
+            except TimeoutException:
+                pass
+
+            WebDriverWait(driver, 20).until(
+                EC.presence_of_element_located(
+                    (By.LINK_TEXT, "Ad Choices")
+                )
+            )
+
+            account_numbers = driver.execute_script(
+                """
+                return Array.from(document.querySelectorAll('li'))
+                    .filter(li => li.outerText.includes('WELLSTRADE'))
+                    .map(li => li.outerText.match(/\\d{4}/)?.[0])
+                    .filter(num => num !== undefined);
+            """
+            )
+            del account_numbers[0]
```

Comment:
This is causing an error for me because I only have valid account numbers in this array. What is this for?

Reply:
This is to delete a duplicate account#. I.E. the list would have [acct1, acct1, acct2, acct3, acct4] returned from that code. This just simply delets the first entry in that list. What error are you getting.

Reply:
![image](https://github.com/user-attachments/assets/aabd9052-4711-4f01-96a1-39bfe8eab55c)
![image](https://github.com/user-attachments/assets/f0d8d8e5-bc7c-47c1-8c5c-69c0c90d566b)

Here is the before and after. It can be seen that I do not get duplicate values, so this makes it to where one of my accounts just never gets selected later on. If we are trying to remove duplicates from the array we need a different way.

Reply:
Interesting. Should be a pretty easy fix, but that is interesting that I get an extra "duplicate" account for some reason and you do not.

Reply:
The latest commit should fix this problem.